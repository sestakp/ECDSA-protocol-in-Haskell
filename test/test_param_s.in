Curve {
p: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEFFFFFC2F
a: 0
b: 7
g: Point {
x: 0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798
y: 0x483ADA7726A3C4655DA4FBFC0E1108A8FD17B448A68554199C47D08FFB10D4B8
}
n: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141
h: 1
}
Key {
d: 0x828f61640d9d585b06d4a45fc986e331ea4e2c81e2af7f9af316c597895736c8
Q: 0x04ab97bfa6a2afa9777647fd394553a1a7b494f817b14529468de1b5247915535240b9acbf058b0389ab35689e98337643002f28f0ca402e88dc25a2c9d41de132
}
Hash: 0x011a775441ecb14943130a16f00cdd41818a83dd04372f3259e3ca7237e3cdaa