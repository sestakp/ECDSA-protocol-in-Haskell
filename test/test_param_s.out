Signature {
   r: 0xd729c9a44d939a041c35c735e665695b8fb34b0b03d22797b60c348fddaecb5f
   s: 0xf934f40db090876c17c562148f1f33c06e0d019e0e4409956aa1f9957a32832a
}